__tests__/app.test.js

```diff
@@ -234,6 +238,138 @@ xdescribe('/api/articles', () => {
 
         })
 
+        test('/api/articles/?comment_count responds 200 with array of objects  ', () => {
```

Comment:
Please don't use comment_count inside the query!

Reply:
It has been removed from tests